tests/test_http_ext_auth.py

```diff
@@ -704,166 +704,231 @@ async def test_http_auth_ext_github_unknown_provider_01(self):
+                url = urllib.parse.urlparse(location)
+                self.assertEqual(url.scheme, server_url.scheme)
+                self.assertEqual(url.hostname, server_url.hostname)
+                self.assertEqual(url.path, f"{server_url.path}/some/path")
+
+                requests_for_token = self.mock_oauth_server.requests[
+                    token_request
+                ]
+                self.assertEqual(len(requests_for_token), 1)
+                body = requests_for_token[0].body
+                assert body is not None
+                self.assertEqual(
+                    json.loads(body),
+                    {
+                        "grant_type": "authorization_code",
+                        "code": "abc123",
+                        "client_id": client_id,
+                        "client_secret": client_secret,
+                        "redirect_uri": f"{self.http_addr}/callback",
+                    },
+                )
 
-            location = headers.get("location")
-            assert location is not None
-            server_url = urllib.parse.urlparse(self.http_addr)
-            url = urllib.parse.urlparse(location)
-            self.assertEqual(url.scheme, server_url.scheme)
-            self.assertEqual(url.hostname, server_url.hostname)
-            self.assertEqual(url.path, f"{server_url.path}/some/path")
+                requests_for_user = self.mock_oauth_server.requests[
+                    user_request
+                ]
+                self.assertEqual(len(requests_for_user), 1)
+                self.assertEqual(
+                    requests_for_user[0].headers["authorization"],
+                    "Bearer github_access_token",
+                )
 
-            requests_for_token = self.mock_oauth_server.requests[token_request]
-            self.assertEqual(len(requests_for_token), 1)
-            body = requests_for_token[0].body
-            assert body is not None
-            self.assertEqual(
-                json.loads(body),
-                {
-                    "grant_type": "authorization_code",
-                    "code": "abc123",
-                    "client_id": client_id,
-                    "client_secret": client_secret,
-                    "redirect_uri": f"{self.http_addr}/callback",
-                },
-            )
+                identity = await self.con.query(
+                    """
+                    SELECT ext::auth::Identity
+                    FILTER .subject = '1'
+                    AND .issuer = 'https://github.com'
+                    """
+                )
+                self.assertEqual(len(identity), 1)
 
-            requests_for_user = self.mock_oauth_server.requests[user_request]
-            self.assertEqual(len(requests_for_user), 1)
-            self.assertEqual(
-                requests_for_user[0].headers["authorization"],
-                "Bearer github_access_token",
-            )
+                # Test Webhook side effect
```

Comment:
This is the new part. The rest of the changes are whitespace changes due to having the `try`/`finally` around this to clean up the webhook config.